src/legacy/core_plugins/kibana/public/visualize/listing/_listing.scss

```diff
@@ -14,4 +14,16 @@
   vertical-align: baseline;
   padding: 0 $euiSizeS;    
   margin-left: $euiSizeS;
-}
\ No newline at end of file
+}
+
+.visListingTable__actionIcon {
+  filter: grayscale(100%);
```

Comment:
This should already be applied if the column is identified as an `ActionsColumn` (er, has action content). I tried winding my way through the files to figure out why this isn't just working, but couldn't track down the root of the issue. Perhaps I'm missing something, but it seems like something is off here.

Reply:
It's because we're using a custom column here, not the standard action, since we want to enable / disable on a per item basis. The selector used by EUI is really specific and involves a 3 class-name chain which doesn't exist in our table here. :/

Reply:
I commented on that in the other PR (we have to get into master, this is getting confusing) - as there are only two consumers of the table list view, we can do this in a clean way, solve it in table list view and don't need this custom column

Reply:
We'll still need a column like this, however, so we'll still end up with our own CSS, I think for the reason I outlined.